data/fixtures/scopes/zig/type.argument.formal.scope

```diff
@@ -0,0 +1,62 @@
+fn foo(aaa: u8, bbb: u8) void {}
+---
+
+[#1 Content] = 0:25-0:29
+                           >----<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Removal] = 0:25-0:30
+                           >-----<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Leading delimiter] = 0:24-0:25
+                          >-<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Trailing delimiter] = 0:29-0:30
+                               >-<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Domain] = 0:0-0:32
+  >--------------------------------<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#1 Insertion delimiter] = " "
+
+
+[#2 Content] = 0:12-0:14
+              >--<
+0| fn foo(aaa: u8, bbb: u8) void {}
+
+[#2 Removal] = 0:12-0:16
+              >----<
+0| fn foo(aaa: u8, bbb: u8) void {}
```

Comment:
The removal range should be `: u8`

https://github.com/cursorless-dev/cursorless/blob/d21522be750a55b70b50ac251300bfd6dd4baace/data/fixtures/scopes/typescript.core/type/type.argument.formal.scope#L10